R/users.R

```diff
@@ -179,3 +179,23 @@ users_exist <- function(vault, users) {
 on_failure(users_exist) <- function(call, env) {
   paste0("Secret ", deparse(call$users), " do not exist")
 }
+
+#' @importFrom openssl fingerprint read_pubkey
+
+lookup_user <- function(key, vault = NULL) {
+  user <- NULL
+  vault <- find_vault(vault)
+  if (is.character(key)) {
```

Comment:
Don't need to handle both cases (file or direct key) here. Just handle the case that actually happens.